builds/misc/images-release.yaml

```diff
@@ -313,18 +313,12 @@ jobs:
           name: "Temperature Sensor"
           project: SimulatedTemperatureSensor
           version: $(version)
-      - bash: |
-            scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration) &&
-            scripts/linux/buildImage.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -i azureiotedge-diagnostics -n $(namespace) -P azureiotedge-diagnostics
-        displayName: 'Build Image - azureiotedge-diagnostics - amd64'
-      - bash: |
-            scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration) --target-arch armv7l &&
-            scripts/linux/buildImage.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -i azureiotedge-diagnostics -n $(namespace) -P azureiotedge-diagnostics --target-arch armv7l
-        displayName: 'Build Image - azureiotedge-diagnostics - ARM32'
-      - bash: |
-            scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration) --target-arch aarch64 &&
-            scripts/linux/buildImage.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -i azureiotedge-diagnostics -n $(namespace) -P azureiotedge-diagnostics --target-arch aarch64
-        displayName: 'Build Image - azureiotedge-diagnostics - ARM64'
+      - template: templates/image-linux.yaml
+        parameters:
+          imageName: azureiotedge-diagnostics
+          name: "Diagnostics Module"
+          project: IotedgeDiagnosticsDotnet
```

Comment:
should the project name be Microsoft.Azure.Devices.Edge.Diagnostics?

Reply:
Its in the same place as SimulatedTemperatureSensor, and I named it the same way